### PR TITLE
Release Pacemaker 3.0.0-rc1

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review this monster? Thanks ...

File: ChangeLog.md [hunk 1]
```diff
@@ -1,3 +1,476 @@
...
+  `with-rsc-role`, and ticket constraints with an invalid `rsc-role` are now
+  ignored (previously, default values would be used)
+* **CIB:** ignore node expressions for meta-attributes
+* **CIB:** treat misconfigured rules and rule expressions as not passing
+* **CIB:** treat negative `migration-threshold` as invalid and use default
+* **CIB:** invalid fencing level indexes are rejected by schema
+* **agents:** drop ocf:pacemaker:o2cb resource agent
+* **agents:** do not pass `HA_mcp`, `HA_quorum_type`, `PCMK_mcp`, or
+  `PCMK_quorum_type`, environment variables to agents
+* **libcrmcommon:** Drop environment variable
```
Comment (clumens) [inline ChangeLog.md:60]: Which environment variable?

File: ChangeLog.md [hunk 2]
```diff
@@ -1,3 +1,476 @@
...
+* **tools:** CIB clients retry signon if first try fails
+* **tools:** don't double-free XML in `crm_verify` after schema update
+
+## Public API changes since Pacemaker-2.1.9
+
+* **libcib:** drop `util_compat.h` header
+* **libcib:** drop enum values `cib_database`, `cib_inhibit_bcast`,
+  `cib_mixed_update`, `cib_quorum_override`, and `cib_zero_copy`
+* **libcib:** drop `cib_api_operations_t` members `delete_absolute()`,
+  inputfd(), `is_master()`, `quit()`, `set_master()`, `set_op_callback()`,
```
Comment (clumens) [inline ChangeLog.md:109]: Ticks around inputfd()

File: ChangeLog.md [hunk 3]
```diff
@@ -1,3 +1,476 @@
...
+  `pe_rsc_needs_quorum`, `pe_rsc_needs_unfencing`, `pe_rsc_notify`,
+  `pe_rsc_orphan`, `pe_rsc_orphan_container_filler`, `pe_rsc_promotable`,
+  `pe_rsc_provisional`, `pe_rsc_reload`, `pe_rsc_replica_container`,
+  `pe_rsc_restarting`, `pe_rsc_runnable`, `pe_rsc_starting`,
+  `pe_rsc_start_pending`, `pe_rsc_stop`, `pe_rsc_stopping`,
+  `pe_rsc_stop_unexpected`, `pe_rsc_unique`, `RSC_ROLE_MASTER_S`,
+  `RSC_ROLE_MAX`, `RSC_ROLE_PROMOTED_LEGACY_S`, `RSC_ROLE_SLAVE_S`,
+  `RSC_ROLE_STARTED_S`, `RSC_ROLE_STOPPED_S`, `RSC_ROLE_UNKNOWN_S`,
+  `RSC_ROLE_UNPROMOTED_LEGACY_S`, and `RSC_ROLE_UNPROMOTED_S`
+* **libpe_status,libpe_rules:** drop functions fail2text(), `pe_pref()`,
```
Comment (clumens) [inline ChangeLog.md:461]: Ticks around fail2text()

File: ChangeLog.md (hunk 1 above)
Comment (kgaillot) [reply ChangeLog.md:60]: oops, copy-paste leftover :)

**Comment by kgaillot** [comment]
updated commit per review

**Review by clumens — APPROVED** [review]

**Pull request merged**